static_src/util/validators.js

```diff
@@ -41,7 +41,7 @@ export function validateString() {
 
 export function validateEmail() {
   return function _validateEmail(value, name) {
-    if (!(/^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/.test(value))) {
+    if (!(/^[a-zA-Z0-9.!#$%&amp;'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/).test(value)) {
```

Comment:
I might make the regex itself a const outside the function so we don't have to create it each time we call the function, and as a bonus we can give it a readable name.